gems/sorbet-runtime/bench/typecheck.rb

```diff
@@ -129,26 +133,6 @@ def self.run
         arg_plus_kwargs(:foo, x: 1, y: 2)
         arg_plus_kwargs(:bar, x: 1)
       end
-
-      time_block(".bind(example).call") do
```

Comment:
Some of the Metrics Rubocops were complaining about this file before I removed these, and while it would probably be fine to just allowlist it, these benchmarks also didn't seem like they were adding much value now that we're done working on `bind_call`.